### PR TITLE
Code review on TorpedoStore class

### PR DESCRIPTION
**Pull request by LiliNemes**
I examined and fixed the problems in the aforementioned class.

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,18 +31,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (LiliNemes) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:34]: ez jó lett, ügyes vagyok :)

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
 * (Deliberately contains bugs.)
 */
 public class TorpedoStore {
+  Random generator = new Random();
```
Comment (Breinich) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:11]: schőőőőőn

**Review by Breinich — CHANGES_REQUESTED** [review]
gut - 10 imsc
adj more imsc

**Review by LiliNemes — COMMENTED** [review]
ok 5 20

**Review by Breinich — APPROVED** [review]

**Pull request merged**